File: semestr1/Lab05/python/zad2.py
import findzero 


def main():
    a = 2
    b = 4
    eps = 1

    for _ in range(1, 10):
        eps = eps/10
        wynik = findzero.findzero(findzero.cos_2, a, b, eps)
        print(f"Dla eps = {eps}, wynik = {wynik}")


if __name__ == "__main__":
    main()

File: semestr1/Lab05/c/zad2.c
#include "findzero.h"
#include "stdio.h"

int main(int argc, char *argv[]){
    // double a,b,eps,wynik;
    // printf("Podaj a: ");
    // scanf("%lf", &a);
    // printf("Podaj b: ");
    // scanf("%lf", &b);
    // printf("Podaj epsilon: ");
    // scanf("%lf", &eps);
    // wynik = findzero(*cos_2,a,b,eps);
    // printf("%lf\n", wynik);
    double a = 2;
    double b = 4;
    double eps = 1;
    double wynik;
    
    for (int i = 1; i < 9; i++){
        eps = eps/10;
        wynik = findzero(cos_2, a,b, eps);
        printf("Dla eps = %.10lf, wynik = %.10lf\n", eps, wynik);
    }
    
}